R/DataProcessing.R

```diff
@@ -292,6 +293,7 @@ readProjectData <- function(fileLines, progress = FALSE, qfeatures = NULL)
   }
 
   ## STN: Disabled. 
+  if(!is.null(qfeatures)){
```

Comment:
Same here, mention that qfeatures==NULL if the function was called from readClusterDataFromProjectFile().
And is an else branch needed ?